### DISCUSSION TITLE
Udemy Videos won't play

### DISCUSSION THREAD

**Discussion by Superpasi7** [Cyfrin/foundry-full-course-cu#456] [General]
Hey everyone I'm using the udemy version of this course and i have an issue where after a few minutes of watching a lecture the video will freeze and won't continue despite internet connection all other lectures from the course won't play either but all other udemy courses that i have work fine so must be something with this course. the issue stops once i complete close my browser (i use google chrome) and all windows of the browser not just udemy but the issue re-appears relatively quickly which makes it impossible for me to go through the course on udemy. Did anyone have the same issue or know what to do about it? Any help is much appreciated.

**Comment by shikhar229169** [comment]
I didn't face this issue.
What you can do is watch the course from youtube and as you cover the lessons, mark them on udemy.
